### PR TITLE
Berserker chads

### PR DESCRIPTION
**Pull request by Refined1**
## About The Pull Request
Re-adds the epic gamer berserker powder and makes the Vex a berserker yet again. Also made meds crafting speed faster.

## Why It's Good For The Game
Legion now has reason to grow shit that isn't better, they also now have something that isn't just healing. Vex is also now more unique as they have Berserker. Meds took too long to make unless you wanted to make the bugged boxes.

## Pre-Merge Checklist

- [ x] You tested this on a local server.
- [ x] This code did not runtime during testing.
- [x ] You documented all of your changes.

## Changelog

:cl:
add: Berserker powder + recipe
add: Berserker loadout for Vex
add: akimbo rocket launcher
add: faster crafting times for meds
/:cl:

### CODE REVIEW

**Comment by Refined1** [comment]
I forgot to mention that it also makes crafting stims and powder way fucking faster so you don't have to stand still for 902 years.

**Comment by Refined1** [comment]
This PR is awesome, merge it! - Tenias (tenias#2089)

**Comment by BadAtThisGame302** [comment]
This PR is awesome, merge it! - Tenias (tenias#2089)

**Comment by Refined1** [comment]
I also made it so you can akimbo rocket launchers because that's what god intended

**Comment by Serxule** [comment]
This PR is awesome, merge it! - serxule (serxule#7334)

**Comment by Legate-Maxson** [comment]
This will not be merged with the akimbo rocket launcher

File: code/modules/fallout/reagents/medicines.dm [hunk 1]
```diff
@@ -192,15 +192,15 @@ datum/reagent/medicine/super_stimpak/on_mob_life(mob/living/M)
 	..()
 	if(isliving(M))
 		to_chat(M, "<span class='notice'>The veil breaks, and the heavens spill out! The spirits of Mars float down from the heavens, and the deafining beat of the holy legion's wardrums fills your ears. Their ethereal forms are guiding you in battle!</span>")
-		M.maxHealth += 20
-		M.health += 20
+		M.maxHealth += 50
```
Comment (EgorDinamit) [inline code/modules/fallout/reagents/medicines.dm:195]: So what, this is just a med-x for Legion now?

**Comment by 5467543** [comment]
legion + med-x
surely this wont be the strongest thing ever implemented :Clueless:

**Comment by Refined1** [comment]
> legion + med-x surely this wont be the strongest thing ever implemented :Clueless:

No, that'd be ghouls. Maybe we'll hit a second place.

File: code/modules/fallout/reagents/medicines.dm (hunk 1 above)
Comment (Refined1) [reply code/modules/fallout/reagents/medicines.dm:195]: Not quite the 70 hp or 95 hp(105 with lifegiver which is 2x the berserker powder) if you take buffout but yeah, essentially.

**Comment by Weaselburg** [comment]
I'm a bit confused. What would the trait do to the vex/cent? Make it so they can use the powder, or does it make it so they always have that trait.

**Comment by Refined1** [comment]
> I'm a bit confused. What would the trait do to the vex/cent? Make it so they can use the powder, or does it make it so they always have that trait.
> 
> Also, it should have the same downsides as med-x.

It does have the same downsides as med-x if not worse. Also no, it won't permanently give them the buff. Berserker powder and berserker martial arts aren't the same.

**Comment by ogrigbe** [comment]
And why are we sneaking in there to make rocket launchers small?....No

**Comment by Refined1** [comment]
I just watched the NCR colonel + HT die to legion without killing anyone. Maybe the NCR needs their OP shit.

**Pull request closed without merging**